### ISSUE TITLE
Expected convergence loss range for Wan fine-tuning loss?

### ISSUE THREAD

**Issue by dazazh** [thu-ml/Motus#18]
First of all, thank you for this amazing repository! It has been very helpful for my research.

I am currently working on fine-tuning the Wan 2.2 based on your codebase. I have adhered to the default configuration and kept the hyperparameters consistent with your original settings (including learning rate, scheduler, etc.).

After training for approximately 5,000 steps, I noticed that the training loss is fluctuating within the range of 0.3 to 0.5. It doesn't seem to be decreasing significantly further.

My Questions:
- Is this loss range (0.3 - 0.5) considered normal or expected for this specific video generation task?
- Based on your experience, what is the typical loss value when the model converges?

**Comment by hanyangyu1021** [comment]
Do you solve the problem? May be we can discuss.

**Comment by strive789** [comment]
@dazazh @hanyangyu1021 

Has this issue been resolved? Could you please explain the cause of it? I have encountered a similar problem in other scenarios, and I would appreciate the opportunity to discuss it further.

**Comment by dazazh** [comment]
> [@dazazh](https://github.com/dazazh) [@hanyangyu1021](https://github.com/hanyangyu1021)
> 
> Has this issue been resolved? Could you please explain the cause of it? I have encountered a similar problem in other scenarios, and I would appreciate the opportunity to discuss it further.

I simply removed the logic in the WAN module that appends action, und, and video tokens for self-attention. Specifically, I reverted **WanSelfAttention** in `Motus/bak/wan/modules/model.py` back to the original version used in WAN.

**Comment by strive789** [comment]
您好，您是说只用wan在RoboTwin上面做视频生成训练，不加入action和und expert，然后video loss在0.3-0.5不下降吗？




         原始邮件
         
       
发件人：Yuhao Zhang（张钰浩） ***@***.***&gt;
发件时间：2026年2月4日 12:52
收件人：thu-ml/Motus ***@***.***&gt;
抄送：strive789 ***@***.***&gt;, Comment ***@***.***&gt;
主题：Re: [thu-ml/Motus] Expected convergence loss range for Wanfine-tuning loss? (Issue #18)



dazazh left a comment (thu-ml/Motus#18)

@dazazh @hanyangyu1021

Has this issue been resolved? Could you please explain the cause of it? I have encountered a similar problem in other scenarios, and I would appreciate the opportunity to discuss it further.

I simply removed the logic in the WAN module that appends action, und, and video tokens for self-attention. Specifically, I reverted WanSelfAttention&nbsp;in Motus/bak/wan/modules/model.py&nbsp;back to the original version used in WAN.

**Comment by dazazh** [comment]
问题应该解决了，这个前向过程应该就是在wan的基础上把action und token拼起来作为qkv，只要把selfattention
block改回原来wan的wanselfattention就能实现纯wan的video
finetune（没有仔细看，这样改完应该和wan2.2是完全对齐的）。之前的问题应该是自己代码哪里没改对，前向和原始的wan前向过程没有对齐，重改以后2w
step loss可以到0.07
